Program.cs

```diff
@@ -1,8 +1,4 @@
-﻿/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Red Hat, Inc. All rights reserved.
- *  Licensed under the MIT License. See LICENSE in the project root for license information.
- *--------------------------------------------------------------------------------------------*/
-using System;
+﻿using System;

```

Comment:
it should contains ASL header as well, no ?

Reply:
No, the filed does not contain a license header in the original code. The ASL license states that we have to preserve existing headers and that we can add additional headers "for our modifications". Both do not apply here.